### ISSUE TITLE
'show' option is not working

### ISSUE THREAD

**Issue by eschreiner** [trentrichardson/jQuery-Impromptu#64]
I try to get rid of the 'slideDown' animation of the prompt, but specifying option show='show' does not work.

**Comment by trentrichardson** [comment]
@eschreiner I think at some point during the re-working for "tour" functionality with states this option became irrelevant.  Ideally this option works as intended, I will try to take a look at correcting it when I get a few minutes free.

**Comment by junkiepipes** [comment]
yeah actually the issue is that the slideDown is now the only option that works.  "show" or "fadeIn" do not change the prompt entrance behaviour.   This is very unfortunate since the slideDown is not a very professional effect and I don't see myself using it unless hopefully Trent can fix it.   Trent... please? :)

**Comment by soulflyman** [comment]
Same problem with `hide: 'hide'`.
A workaround, at least for the 'show' bug, is to set overlayspeed and promptspeed to 0.
